static/src/javascripts/projects/common/modules/email/email-article.js

```diff
@@ -0,0 +1,283 @@
+        },
+        headline: 'Here’s the real story',
+        description:
+            'Lose yourself in a great story. From politics to psychology, food to technology, culture to crime – the best stories, the biggest ideas, the arguments that matter.',
+        successHeadline: 'Thank you for signing up to The Long Read',
+        successDescription: 'You’ll receive an email every weekend.',
+        insertMethod: insertBottomOfArticle,
+    },
+    bookmarks: {
+        listId: '3039',
+        listName: 'bookmarks',
+        campaignCode: 'bookmarks_article_bottom',
+        displayName: {
+            normalText: 'book',
+            accentedText: 'marks',
+        },
+        headline: 'Bookmarks: read me first',
+        description:
+            'Sign up for our weekly email for book lovers and discover top 10s, expert book reviews, author interviews, and enjoy highlights from our columnists and community every weekend.',
+        successHeadline: 'Thank you for signing up to Bookmarks',
+        successDescription: 'You’ll receive an email every weekend.',
+        insertMethod: insertBottomOfArticle,
+    },
+    greenLight: {
+        listId: '38',
+        listName: 'greenLight',
+        campaignCode: 'green_light_article_bottom',
+        displayName: {
+            normalText: 'green',
+            accentedText: 'light',
+        },
+        headline: 'The most important stories on the planet',
+        description:
+            'Sign up to Green Light for environment news emailed direct to you every Friday. And besides the week’s biggest stories and debates, you can expect beautifully curated wildlife galleries, absorbing podcasts and eco-living guides.',
+        successHeadline: 'Thank you for signing up to Green Light',
+        successDescription: 'You’ll receive an email every Friday.',
+        insertMethod: insertBottomOfArticle,
+    },
+    theGuardianToday: {
+        listId: (() => {
+            switch (config.get('page.edition')) {
+                default:
+                    return '37';
+
+                case 'US':
+                    return '1493';
+
+                case 'AU':
+                    return '1506';
+            }
+        })(),
+        listName: 'theGuardianToday',
+        campaignCode: 'guardian_today_article_bottom',
+        displayName: {
+            normalText: 'theguardian',
+            accentedText: 'today',
+        },
+        headline: 'The headlines, the analysis, the debate',
+        description:
+            'Get the whole picture from a source you trust, emailed to you every morning. The biggest stories examined, and diverse, independent views &ndash; the Guardian Today delivers the best of our journalism.',
+        successHeadline: 'Thank you for signing up to the Guardian Today',
+        successDescription:
+            'We will send you our picks of the most important headlines tomorrow morning.',
+        insertMethod: insertBottomOfArticle,
+    },
+};
```

Comment:
We could probably define a type for `listConfig`s

Reply:
good call

Reply:
done 👍